modules/portal/portal-api/src/main/java/com/enonic/xp/portal/url/ApiUrlParams.java

```diff
@@ -46,9 +46,16 @@ public ApiUrlParams api( final String value )
         return this;
     }
 
-    public ApiUrlParams path( final List<String> value )
+    public ApiUrlParams path( final Object value )
     {
-        this.path = value;
+        if ( value instanceof ScriptValue && ( (ScriptValue) value ).isArray() )
```

Comment:
it is work of ApiUrlHander.
Also one pathSegments should accept List, another path - accepts path